dashboard/app/models/learning_goal_ai_evaluaton.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: learning_goal_ai_evaluatons
+#
+#  id               :bigint           not null, primary key
+#  user_id          :integer
+#  learning_goal_id :integer
+#  prompt_version   :datetime
```

Comment:
this is interesting, @wilkie or @bethanyaconnor have you given any thought to how we might implement prompt versioning? this makes sense to me if the prompt is going to be a row in a table with a created_at timestamp. If we store it in S3 then an S3 version (whatever that looks like) might be more appropriate.

unless a plan is clear, perhaps it would be a little cleaner to leave this out of the table for now, and we can add it when we figure out how we'll do prompt versioning?

Reply:
I'll let Wilkie chime in here -- I think this is from the tech spec and I haven't given any thought to what this might look like

Reply:
my vote is to omit this field unless / until we are sure we know what we want it to look like, because adding a field is easier than changing an existing one (or if you know of a way to change a field which ignores/destroys existing data then could work too)